contact.php

```diff
@@ -107,8 +110,6 @@
         $subject = 'movie pass signup';
 
         $body = "From: $name E-Mail: $email Message: $message";
```

Comment:
This looks like it could use some formatting, I'm not sure that would look very nice to the end user